README.md

```diff
@@ -1,64 +1,109 @@
+<img alt="openBalena" src="docs/assets/openbalena-logo.svg" height="82">
 
-## Host requirements
+---
 
-- Docker >= 18.05.0
-- Docker Compose >= 1.11
-- OpenSSL >= 1.0.0
-- Python >= 2.7 or >=3.4
-- NodeJS >= 4.0
+OpenBalena is a platform to deploy and manage connected devices. Devices run
+[balenaOS][balena-os-website], a host operating system designed for running
+containers on IoT devices, and are managed via the [balena CLI][balena-cli],
+which you can use to configure them, push updates, check status, view logs,
+and so forth. Its backend services, composed of battle-tested components that
+[we’ve run in production][balena-cloud-website] for years, can store device
+information securely and reliably, allow remote management via a built-in VPN
+service, and efficiently distribute container images to your devices.
 
-## Installation
+To learn more about openBalena, visit [balena.io/open][open-balena-website].
 
-Make sure you have the software listed above installed.
 
-In a terminal, clone the project with:
+## Features
 
-    $ git clone https://github.com/balena-io/open-balena.git
+- **Simple provisioning**: Adding devices to your fleet is a breeze
+- **Easy updates**: Remotely update the software on your devices with a single command
+- **Container-based**: Benefit from the power of virtualization, optimized for the edge
+- **Scalable**: Deploy and manage one device, or one million
+- **Powerful API & SDK**: Extend openBalena to fit your needs
+- **Built-in VPN**: Access your devices regardless of their network environment
 
-Change into the `open-balena` directory and run the configuration script.
-This will create a new directory, `config`, and generate appropriate SSL
-certificates and configuration for the instance.
 
-    $ ./scripts/quickstart
+## Roadmap
 
-You may optionally configure the instance to run under a custom domain name.
-The default is `openbalena.local`. For example:
+openBalena is currently in beta. While fully functional, it lacks features we
```

Comment:
Since we capitalize `BalenaCloud` when it's in the beginning of a sentence, should we do the same for `OpenBalena`?
```suggestion
OpenBalena is currently in beta. While fully functional, it lacks features we
```